extensions/2.0/Khronos/KHR_materials_volume/README.md

```diff
@@ -0,0 +1,192 @@
+
+<figure style="text-align:center">
+<img src="./figures/volume.svg"/>
+<figcaption>Object with homogeneous volume and an index of refraction of 1.5.</figcaption>
+</figure>
+
+## Extending Materials
+
+The volumetric properties are defined by adding the `KHR_materials_volume` extension to any glTF material.
+
+```json
+materials: [
+    {
+        "extensions": {
+            "KHR_materials_volume": {
+                "sigmaT": [ 78.03, 17.83, 126.29 ],
+                "albedo": [ 0.29, 0.82, 0.99 ],
+                "anisotropy": 0.0
+            }
+        }
+    }
+]
+```
+
+Note that there are no textures in this extension, the medium is homogeneous. If a texture is defined in `KHR_materials_ior`, it is ignored if `KHR_materials_volume` is active.
+
+## Properties
+
+The extension provides two parameters to describe the medium and one parameter to control the phase function.
+
+| | Type | Description | Required |
+|-|------|-------------|----------|
+| **sigmaT** | `number[3]` | Attenuation coefficient (also called extinction coefficient) in inverse meters. | No, default: `[0.0, 0.0, 0.0]` |
+| **albedo** | `number[3]` | Single-scattering albedo. | No, default: `[0.0, 0.0, 0.0]` |
+| **anisotropy** | `number` | Anisotropy of the phase function in range [-1, 1] | No, default: `0` |
+| **thicknessTexture** | `textureInfo` | A grayscale texture that defines the thickness of the volume beneath the surface. May be used by renderers that are not able to derive the thickness from the actual geometry, otherwise it should be ignored. | No |
+| **minThickness** | `float` | When using a thickness texture, a texture value of 0.0 is interpreted to be `minThickness`. When a constant thickness is used, the thickness will be `minThickness`. | No, default: `0` |
+| **maxThickness** | `float` | When using a thickness texture, a texture value of 1.0 is interpreted to be `maxThickness`. Values between 0.0 and 1.0 are linearly scaled between minThickness and `maxThickness`. | No, default: `1` |
+
+### Conversions
+
+Alternatively, subsurface scattering can be parametrized with absorption coefficient **sigmaA** and scattering coefficient **sigmaS**. These are converted as follows:
+
+```
+sigmaT = sigmaS + sigmaA
+albedo = sigmaS / (sigmaS + sigmaA)
+```
+
+Sometimes the mean free path length **mfp** is given:
+
+```
+sigmaT = 1 / mfp
+```
+
+The attenuation color is a 3-channel value that goes up to infinity. By splitting it into two values, an RGB attenuation color and a distance, the color is easier to set up. In the following equation, **attenuationColor** defines the remaining, wavelength-dependent energy of a ray of white light after travelling a certain distance (**attenuationDistance**) through the medium.
```

Comment:
As I understood AttenuationColor' is actually transmittance of the ray after traveling attenuationDistance through the medium. I think Transmittance is a better name because from 'AttenuationColor' it's bit ambiguous if it is attenuated or remaining part of the energy.